forem/forem
# [Admin Tooling] Allow admins delete empty Connect Channels

## What type of PR is this? (check all applicable)


## Description
This PR allows admins to delete channels that have no users from `/admin/chat_channels`.

## QA Instructions, Screenshots, Recordings
1. Navigate to `localhost:300/admin/chat_channels` as an `:admin` or `:super_admin` user
2. If there are channels with users, you should see the text _**"Cannot delete; channel has users"**_ under their "Delete Channel" column
3. To create a user-less channel, create a test channel at the top. Your created channel should appear in the list, and its `Delete Channel` button should be visible
4. Delete your test channel to observe functionality

https://www.loom.com/share/ea26f5ddfd394faaab7dcd56eb45d78e


## Added tests?

- [X] yes, and quite proud of them 😊 

## Added to documentation?


![Girl collapsing on floor](https://media.giphy.com/media/LQ3n4TeWatrcLy9zQ1/giphy-downsized.gif)
